src/test/java/healthy/lifestyle/backend/data/DataUtil.java

```diff
@@ -94,6 +94,7 @@ public SignupRequestDto createSignupRequestDto(String seed, Long id) {
                 .confirmPassword("password-" + seed)
                 .fullName("Full Name " + seed)
                 .countryId(id)
+                .age(20)
```

Comment:
Let's specify age as a method  parameter - `createSignupRequestDto(String seed, Long countryId, Integer age)`. `DataHelper` class should also be refactored with a respect to an age field.